client-spark/spark2/src/main/java/org/apache/spark/shuffle/writer/RssShuffleWriter.java

```diff
@@ -120,6 +123,17 @@ public RssShuffleWriter(
     this.shuffleWriteClient = shuffleWriteClient;
     this.shuffleServersForData = rssHandle.getShuffleServersForData();
     this.partitionToServers = rssHandle.getPartitionToServers();
+    setMemoryShuffleEnabled(sparkConf.get(RssClientConfig.RSS_STORAGE_TYPE));
+  }
+
+  private void setMemoryShuffleEnabled(String storageType) {
```

Comment:
isMemoryShuffleEnabled = getMemoryShuffleEnabled(storageType); 
may be better?

Reply:
ok